Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM scratch
+FROM ubuntu:14.04
```

Comment:
i already tried it with alpine and it doesn't work. Also busybox does not work as i always get `ash: docker: not found` with the same setup.